addons/l10n_mu_account/models/res_company.py

```diff
@@ -0,0 +1,17 @@
+from markupsafe import Markup
+
+from odoo import _, api, fields, models
+
+
+class BaseDocumentLayout(models.TransientModel):
+    _inherit = 'base.document.layout'
+
+    @api.model
+    def _default_company_details(self):
```

Comment:
for some reason, when looking at the document, I can't see the company registry for my own company. What am I missing ? Is there a condition I miss ?